### ISSUE TITLE
Lost Legends book in the College of Winterhold Arcanaeum breaks the Forbidden Legend (Gauldur Amulet) quest.

### ISSUE THREAD

**Issue by Durgenage** [Guitarninja2/septimus#422]
If you pick up this specific copy of Lost Legends BEFORE reading any other copy, it will (as far as I know) permanently prevent you from beginning the Forbidden Legend questline. Reading another copy of the book will not trigger the quest, and the Gauldur Fragments and Writs of Sealing on the Gauldursson Draugr Bosses (the other way of starting the quest) do not appear.
![Untitled](https://user-images.githubusercontent.com/96037567/162665092-650b5ef0-e3f0-4904-af3a-fbc84f096a3b.png)

**Comment by Xgatt** [comment]
This one sounds a bit strange. Can you do a bit more testing on this? First, this quest has a lot of known bugs in vanilla itself: https://elderscrolls.fandom.com/wiki/Forbidden_Legend#Bugs. Usually picking up a copy of the book should fix the script.

Look over those and see if any of those apply to your save here. I don't see anything different about this book than any other version of the book, since it's just a ref to the same book as every other copy. Second, can you test it on a new save to confirm it's this specific copy causing issues?

**Comment by Guitarninja2** [comment]
Yeah this one is super weird... it works ok for me but others have reported it breaking.

**Comment by Guitarninja2** [comment]
Yeah I'm going to leave this open and we can take a look when 3.0.4 comes out
